docs/tempo/website/api_docs/_index.md

```diff
@@ -191,7 +191,8 @@ $ curl -G -s http://localhost:3200/api/search --data-urlencode 'tags=service.nam
 
 ### Search Tags
 
-<span style="background-color:#f3f973;">This experimental endpoint is disabled by default and can be enabled via the `search_enabled` YAML config option.</span>
+<span style="background-color:#f3f973;">This experimental endpoint is disabled by default and can be enabled via the `search_enabled` YAML config option.
+Ingester configuration `complete_block_timeout` affects how long tags are available for search.</span>
```

Comment:
thanks for this addition, but we should move the new line

> Ingester configuration `complete_block_timeout` affects how long tags are available for search.

outside of the experimental warning.

Reply:
I agree with Joe. The configuration isn't part of the warning per se.